browse/services/document/author_affil.py

```diff
@@ -102,7 +106,7 @@ def _parse_author_affil_split(author_line: str)->Dict:
                  r'^(.*)\s+(' + PREFIX_MATCH + r')\s(' +
                  PREFIX_MATCH + r')\s(\S+)$'),
                 ('name-prefix-name',
-                 r'^(.*)\s+(' + PREFIX_MATCH + ')\s(\S+)$'),
+                 r'^(.*)\s+(' + PREFIX_MATCH + r')\s(\S+)$'),
```

Comment:
Was the lack of the the r here causing errors or is this a style change?

Reply:
I think all this does is silence the linter, which just complains about invalid escape sequences in the string.